apps/ledger-live-mobile/src/components/RootNavigator/SyncOnboardingNavigator.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { useMemo } from "react";
+import { useTheme } from "styled-components/native";
+import { createStackNavigator } from "@react-navigation/stack";
+
+import { getStackNavigatorConfig } from "../../navigation/navigatorConfig";
+import { ScreenName } from "../../const";
+import { SyncOnboarding } from "../../screens/SyncOnboarding/index";
+import CompletionScreen from "../../screens/SyncOnboarding/CompletionScreen";
+import { SyncOnboardingCompanionParams } from "../../screens/SyncOnboarding/types";
+
+export type SyncOnboardingStackParamList = {
```

Comment:
Declaring this in this file creates a require cycle no? I'd put this in a separate file like `types.ts`
For instance this file imports from `CompletionScreen.tsx` and `CompletionScreen.tsx` imports the params from this file.

Reply:
It would create a require cycle if we do a simple `import { SyncOnboardingStackParamList }` in `CompletionScreen` yes. But if we do a `import type { SyncOnboardingStackParamList }` we're good ;) 

And that's what we did normally

Reply:
Oh ok good to know!